.github/CHANGELOG.md

```diff
@@ -2,60 +2,14 @@
 
 <h3>New features since last release</h3>
 
-* It is now possible to specify custom wire labels, such as `['anc1', 'anc2', 0, 1, 3]`, where the labels
-  can be strings or numbers. For this, pass a list to the wires argument when creating the device:
-  
-  ```pycon
-  >>> dev = qml.device("default.qubit", wires=['anc1', 'anc2', 0, 1, 3])
-  ```
-  The quantum operations are now called with the custom wire labels:
-    
-  ``` python
-  >>> @qml.qnode(dev)
-  >>> def circuit():
-  ...    qml.Hadamard(wires='anc2')
-  ...    qml.CNOT(wires=['anc1', 3])
-  ...    ...
-  ```
-  The existing behaviour, in which the number of wires is specified on device initialization,
-  continues to work as usual.   
-  
-  ```pycon
-  >>> dev = qml.device("default.qubit", wires=5)
-  ``` 
-  [(#666)](https://github.com/XanaduAI/pennylane/pull/666)
- 
-* Adds a device test suite, located at `pennylane/devices/tests`, which can be used 
-  to run generic tests on core or external devices calling 
-  
-  ```pycon
-  >>> pytest pennylane/plugins/tests --device default.qubit --shots 1234 --analytic False
-  ``` 
-  The command line arguments are optional.
-
-  * If `--device` is not given, the tests are run on the core devices that ship with PennyLane.
-
-  * If `--shots` is not given, a default of 10000 is used. The shots argument is ignored for devices running in
-    analytic mode.
-
-  * If `--analytic` is not given, the device's default is used.
-
-  Other arguments of the device, such as `qiskit.aer`'s compulsory `backend_options`,
-  can be defined in the `config.toml` file containing custom PennyLane configurations.
-
-  If the tests are run on external devices, the device and its dependencies must be
-  installed locally.
+<h4>New and improved simulators</h4>
```

Comment:
I know it's a separate repo, but we should add pennylane.lightning here

Reply:
Good idea!